cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/templates/common/slurm_utils.conf

```diff
@@ -0,0 +1,7 @@
+{%- macro node_name_section(queue_name, compute_resource_name, size, is_static) -%}
```

Comment:
what about naming the file `slurm_parallelclulster_utils.conf`, so that it's aligned with other files?